cli/src/main/java/org/jboss/as/cli/impl/aesh/cmd/deployment/AbstractUndeployCommand.java

```diff
@@ -118,10 +118,13 @@ static void undeployName(CommandContext ctx, String name,
         try {
             ModelNode request = buildRequest(ctx,
                     name, allServerGroups, serverGroups, keepContent, headers);
-            final ModelNode result = ctx.
-                    getModelControllerClient().execute(request);
-            if (!Util.isSuccess(result)) {
-                throw new CommandException(Util.getFailureDescription(result));
+
+            if (request.get(Util.STEPS).isDefined()) {
```

Comment:
This is really what is fixing the standalone case. I would say that this is the only required change to align standalone and domain. There is no problem to disable an already disabled deployment, CLI command allows to do that.